pdns/pdnsutil.cc

```diff
@@ -1682,7 +1682,7 @@ static bool showZone(DNSSECKeeper& dk, const DNSName& zone, bool exportDS = fals
         cout<<"ID = "<<value.second.id<<" ("<<DNSSECKeeper::keyTypeToString(value.second.keyType)<<")";
       }
       if (value.first.getKey()->getBits() < 1) {
-        cerr<<" <key missing or defunct>" <<endl;
+        cout<<" <key missing or defunct>" <<endl;
```

Comment:
I wonder if the move from `stderr` to `stdout` really makes sense?

Reply:
During regression tests, the output of pdnsutil is passed through `grep` and friends to get key IDs, and this message prints to the terminal repeatedly with no context. Changing to `cout` avoids that. In general I'm not sure it ever makes sense to mix stdout and stderr in the same output line.

Does that help?

Reply:
Yes and no! I'm now wondering if there are cases where this message is actually helpful? :)

Reply:
Perhaps if we add `run pdnsutil hsm create-key?` to it, I'll do that :)

Reply:
pushed that